cluster/gce/trusty/kube-manifests/kube-apiserver.manifest

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
@dchen1107 , I add a TODO in build/common.sh for getting rid of using separate manifest copies in trusty. After this PR is merged for 1.2, I will make another PR to address it, either cherry-picking it for 1.2 or for the next patch release of 1.2